src/main/scala/firrtl/Visitor.scala

```diff
@@ -139,6 +139,8 @@ class Visitor(infoMode: InfoMode) extends FIRRTLBaseVisitor[FirrtlNode] {
           case "SInt" => if (ctx.getChildCount > 1) SIntType(IntWidth(string2BigInt(ctx.IntLit.getText)))
           else SIntType(UnknownWidth)
           case "Clock" => ClockType
+          case "Analog" => if (ctx.getChildCount > 1) AnalogType(IntWidth(string2BigInt(ctx.IntLit.getText)))
+          else AnalogType(UnknownWidth)
```

Comment:
This can be:
```
case "Analog" if (ctx.getChildCount > 1) => AnalogType(IntWidth(string2BigInt(ctx.IntLit.getText)))
case "Analog" => AnalogType(UnknownWidth)
```
But you don't need to fix it :)